content/docs/user-guide/experiment-management/running-experiments.md

```diff
@@ -0,0 +1,569 @@
+
+```dvc
+$ dvc exp run
+```
+
+If all dependencies and the experiment output are not changed and available in
+the workspace, the `dvc exp run` doesn't rerun the commands.
+
+For a pipeline composed of `extract`, `transform`, `train`, `evaluate`, if a
+dependency of `train` stage has changed, the dependent stages (`evaluate`) are
+also run.
+
+### Specifying Targets
+
+By default `dvc exp run` uses `dvc.yaml` file in the current directory. You can
+specify other directories or pipeline elements to run. These are specified as
+the last element of the command.
+
+Pipelines can be of the following:
+
+- A particular `dvc.yaml` file: `dvc exp run my-project/dvc.yaml`
+- A directory that contains a `dvc.yaml` file: `dvc exp run my-project`
+- A stage from the default `dvc.yaml` file: `dvc exp run extract`
+- A stage from a specific `dvc.yaml` file:
+  `dvc exp run my-project/dvc.yaml:extract`
+- A set of stages from the default `dvc.yaml` file:
+  `dvc exp run --glob 'train-*'`
+
+### Running a Single Stage
+
+In some cases you may need to run a single stage in the pipeline. The
+`--single-item` (`-s`) flag allows to run the associated command of a single
+stage.
+
+If the pipeline has `extract`, `transform`, `train`, `evaluate` stages and you
+only want to run the transform stage to check its outputs, you can do so by:
+
+```dvc
+$ dvc exp run --single-stage transform
+```
+
+### Running all Pipelines
+
+In larger projects, there may be more than a single `dvc.yaml` file that
+contains multiple pipelines. In this case, you can run all pipelines with a
+single command.
+
+```dvc
+$ dvc exp run --all-pipelines
+```
+
+Note that the order to run these pipelines is not specified. If you have a
+pipeline in `my-dir/dvc.yaml` and `another-dir/dvc.yaml`, either of these
+pipelines can be run first.
+
+### Interactive Reproduction
+
+When you want to have more granular control over which stages are run, you can
+use `--interactive` flag. This allows you to confirm each stage before running.
+
+```dvc
+$ dvc exp run --interactive
+Going to reproduce stage: 'train'. Are you sure you want to continue? [y/n]
+```
+
+### Recursive Search for Pipelines
+
+If you have a directory tree containing multiple pipelines and want to reproduce
+all starting from a root directory, you can do so by `--recursive` (`-R`) flag.
+For a directory tree, it starts from the root and descends into subdirectories.
+
+```dvc
+$ dvc exp run -R my-sub-project/
+```
+
+The above command begins from `my-sub-project/dvc.yaml`, and proceeds to
+`my-sub-project/sub-module/...` to run all the pipelines.
+
+### Improvements over `dvc repro`
+
+The classical way of running the pipeline is using `dvc repro`. DVC introduced
+`dvc exp` as an _experiment management_ feature in version 2.0. _Experiment
+management_ involves more than executing the pipelines.
+
+In the following sections, we describe the features that make `dvc exp run` more
+suitable for machine learning experimentation.
```

Comment:
Good note but probably not needed or maybe just as a note (md quote block) somewhere else. I think we just need the "New in DVC 2.0" line at the top of this (and all Exp Mgmt) guide(s). Like in https://dvc.org/doc/command-reference/exp

Reply:
Users of former versions may ask "what's the difference between this and that?" Also someone just learning about DVC and had met `dvc repro` just a few pages before may wonder why there are two commands doing a similar job.

Reply:
This can be a summary of the differences from `dvc repro` but I'm not sure. Keeping this as is.

Reply:
I don't see how telling readers the history of when each command was developed and why is helpful. This isn't about different versions or improvements either since both commands exist at the same time and behave the same way (one is not better than the other).

I think bringing this up has the potential to confuse more people than those who happened to wonder about this. Also, the explanation given "Experiment management_ involves more than executing the pipelines" doesn't explain what are the improvements.

Reply:
We are not telling the history here, there are two (or including `dvc run`, three) commands that run the pipeline and I believe it's a common topic to discuss the differences.  Why DVC team bothered to add another command that seemingly does the same thing? When I should use one and not the other?

Reply:
Valid discussion but the guide should focus on running experiments. With the goal of moving on I think I'll remove this or make it into a note somewhere for now... ⏳

Reply:
Done in fd1d1c5f.